src/components/Layout/index.js

```diff
@@ -2,7 +2,7 @@ import Head from 'next/head';
 import Navbar from '../Navbar';
 import Footer from '../Footer';
 
-export default function Layout({ children }) {
```

Comment:
@rabeehrz props are being destructured here!

Reply:
Oh, so shouldn't you destructure them while using arrow functions as well?

Reply:
> Oh, so shouldn't you destructure them while using arrow functions as well?

destructuring them in the next commit.